README.md

```diff
@@ -150,6 +156,26 @@ GRAFANA_TEST_DB=mysql go test ./pkg/...
 GRAFANA_TEST_DB=postgres go test ./pkg/...
 ```
 
+#### End to end
+
+Execute all end to end tests
+
+```bash
+yarn e2e-tests
+```
+
+Execute all end to end tests using using a specific url
```

Comment:
I believe they usually spell it as "end-to-end", but that's not important really :)

Reply:
Done